### PR TITLE
REST - mark up 'description' recursively

### PR DESCRIPTION
**Pull request by hellosnow**
mark up `description` recursively

@chenkennt @superyyrrzz @vwxyzh @ansyral @qinezh

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.RestApi/BuildRestApiDocument.cs [hunk 1]
```diff
@@ -4,22 +4,23 @@
...
     using Microsoft.DocAsCode.Utility;
-    using Microsoft.DocAsCode.Utility.EntityMergers;
+
+    using Newtonsoft.Json.Linq;
 
     [Export(nameof(RestApiDocumentProcessor), typeof(IDocumentBuildStep))]
     public class BuildRestApiDocument : BaseDocumentBuildStep
     {
-        private readonly ReflectionEntityMerger Merger = new ReflectionEntityMerger();
+        private static readonly HashSet<string> MarkupKeys = new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "description" };
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.RestApi/BuildRestApiDocument.cs]: ignore case?

**Comment by superyyrrzz** [comment]
:shipit:

**Comment by qinezh** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.RestApi/BuildRestApiDocument.cs (hunk 1 above)
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.RestApi/BuildRestApiDocument.cs]: If ignore case, success to pass in [http://bigstickcarpet.com/swagger-parser/www/index.html](http://bigstickcarpet.com/swagger-parser/www/index.html) while failed in swagger io, better change to case sensitive.

---
In reply to: [86910967](https://github.com/dotnet/docfx/pull/864#discussion_r86910967) [](ancestors = 86910967)

**Pull request merged**